sph/include/sph/hydro_ve/momentum_energy_kern.hpp

```diff
@@ -62,7 +62,7 @@ HOST_DEVICE_FUN T avRelVelCorrection(util::array<Tc, 3> R, Tc dist, Th hi, Th hj
     T A_ab = T(0);
 
     if (dmy2 != T(0)) { A_ab = dmy1 / dmy2; }
-    T eta_ab = stl::min(dist / hi, dist / hj);
+    T eta_ab = dist * stl::min(hiInv, hjInv);
```

Comment:
You can pass directly dist/hi and dist/hj and avoid passing one of the three. Just pass v1 and v2, which are the typical name for the scaled distances between particles, and are already calculated when calling the kernel a few lines before.